hms-api/src/service/participant-service.ts

```diff
@@ -110,7 +110,6 @@ export async function removeParticipant(
   id: Uuid,
 ): Promise<ParticipantDeleteResponse> {
   try {
-    await removeIdeasForOwner(id);
```

Comment:
why did this go away? It also seems really unrelated to the comments stuff

Reply:
unsure if we really should delete all ideas if a user leaves a hackathon...more like when the user gets deleted or something like that and even then I would argue that we should just set the user to "anonymus" and keep the idea but thats more like a personal opinion

Reply:
yeah I'd be ok with that as well, but we need to do something.  If we do nothing then we end up with a lot of broken data (user IDs in the Ideas that point to nothing)

Reply:
probably worth a seperate ticket to have a solution for everything when a users gets deleted